Documentation/teaching/index.rst

```diff
@@ -39,7 +39,6 @@ then point your browser at **Documentation/output/labs/index.html**.
    lectures/syscalls.rst
    lectures/interrupts.rst
    lectures/smp.rst
-   lectures/debugging.rst
```

Comment:
This looks like it was removed by mistake.

Reply:
My bad.